web-components/src/rootStyles/style.css

```diff
@@ -4,15 +4,18 @@
 
 /* For more infos about the font and its settings check out: https://fontsource.org/fonts/open-sans/cdn */
 @font-face {
-  font-family: "Open Sans";
-  font-weight: 100 900;
+  font-family: "Inter";
+  font-style: normal;
   font-display: swap;
-  src: url(https://cdn.jsdelivr.net/fontsource/fonts/open-sans:vf@latest/latin-wght-normal.woff2) format('woff2-variations');
-  unicode-range: U+0000-00FF,U+0131,U+0152-0153,U+02BB-02BC,U+02C6,U+02DA,U+02DC,U+0300-0301,U+0303-0304,U+0308-0309,U+0323,U+0329,U+2000-206F,U+2074,U+20AC,U+2122,U+2191,U+2193,U+2212,U+2215,U+FEFF,U+FFFD;
+  font-weight: 100 900;
+  src: url(https://cdn.jsdelivr.net/fontsource/fonts/inter:vf@latest/latin-wght-normal.woff2)
```

Comment:
Interesting that the name is `latin-wght-normal`. this is the variable font though, right?

Reply:
yes, the `vf` is for "variable font" afaict, I have that same change locally on my machine from when I showed it off at a presentation :D

Reply:
Weird :)